src/proxy-ui-api/frontend/src/views/KeysAndCertificates/SignAndAuthKeys/SignAndAuthKeys.vue

```diff
@@ -24,40 +24,38 @@
         :isOpen="isExpanded(token.id)"
       >
         <template v-slot:action>
-          <large-button
-            @click="login(token, index)"
-            v-if="!token.logged_in"
-            :disabled="!token.available"
-          >{{$t('keys.logIn')}}</large-button>
-          <large-button
-            @click="logout(token, index)"
-            v-if="token.logged_in"
-            outlined
-          >{{$t('keys.logOut')}}</large-button>
+          <template v-if="canActivateToken">
+            <large-button
+              @click="login(token, index)"
+              v-if="!token.logged_in"
+              :disabled="!token.available"
+            >{{$t('keys.logIn')}}</large-button>
+            <large-button
+              @click="logout(token, index)"
+              v-if="token.logged_in"
+              outlined
+            >{{$t('keys.logOut')}}</large-button>
+          </template>
+          <div v-else></div>
```

Comment:
I guess can be removed

Reply:
I modified the expandable component so that I was able to remove this.